SoftLayer/CLI/virt/notifications.py

```diff
@@ -12,7 +12,7 @@
 @click.argument('identifier')
 @environment.pass_env
 def cli(env, identifier):
-    """Get all hardware notifications."""
+    """Get all VS notifications."""
```

Comment:
Maybe is better "virtual server" but "vs" is also fine

Reply:
I used `VS` because is used many commands